committeeoversightapp/urls.py

```diff
@@ -1,11 +1,11 @@
 from django.conf.urls import url, include
 from committeeoversightapp.views import EventCreate, EventList, EventDelete, \
-                                        EventEdit, EventListJson
+                                        EventEdit, EventListJson, EventDetail
 
 urlpatterns = [
     url(r'accounts/', include('django.contrib.auth.urls')),
     url(r'^hearing/create/$', EventCreate.as_view(), name='create-event'),
-    url(r'^hearing/list/$', EventList.as_view(), name='list-event'),
+    url(r'^hearing/view/(?P<pk>.*?/.*?)/', EventDetail.as_view(), name="detail-event"),
```

Comment:
I find these kinds of regexes hard to grok. Is there a reason you prefer `url()` with a raw regex to [`path()`](https://docs.djangoproject.com/en/2.2/ref/urls/#path) with converters?

Reply:
This is just the convention I started following... some of the OCD urls are wonky, since their IDs includes slashes, but let me see if I can refactor.

Reply:
Ok! I see that `url` is slated for deprecation, so I've converted all urls in this project to `path` or `re_path`. I don't think it's worth taking more time to figure out how to do this without regex.

Reply:
Seems fine to me, I just wanted to make sure you were aware of `path()` since it offers a simplified API.